### PR TITLE
Extend CpuExecutionProvider to support double on ceil.

### PR DESCRIPTION
**Pull request by WilBrady**
While expanding eager mode op support, I stumbled upon an issue with python print of an ort tensor of type double where it calls ceil with type double. This print fails for the cpu execution provider. To address this, I am extending ceil support to double.

### CODE REVIEW

**Review by souptc — APPROVED** [review]

File: orttraining/orttraining/eager/ort_aten.cpp [hunk 1]
```diff
@@ -1129,7 +1129,6 @@ at::Tensor& mm_out(
   return out;
```
Comment (jamill) [inline orttraining/orttraining/eager/ort_aten.cpp:1129]: If there are no functional changes to this file, can we remove this change?

File: onnxruntime/core/providers/cpu/cpu_execution_provider.cc [hunk 2]
```diff
@@ -66,6 +66,7 @@ class ONNX_OPERATOR_VERSIONED_TYPED_KERNEL_CLASS_NAME(kCpuExecutionProvider, kOn
 class ONNX_OPERATOR_VERSIONED_TYPED_KERNEL_CLASS_NAME(kCpuExecutionProvider, kOnnxDomain, 6, 12, uint64_t, Abs);
 class ONNX_OPERATOR_VERSIONED_TYPED_KERNEL_CLASS_NAME(kCpuExecutionProvider, kOnnxDomain, 6, 12, float, Floor);
 class ONNX_OPERATOR_VERSIONED_TYPED_KERNEL_CLASS_NAME(kCpuExecutionProvider, kOnnxDomain, 6, 12, float, Ceil);
+class ONNX_OPERATOR_VERSIONED_TYPED_KERNEL_CLASS_NAME(kCpuExecutionProvider, kOnnxDomain, 6, 12, double, Ceil);
```
Comment (jamill) [inline onnxruntime/core/providers/cpu/cpu_execution_provider.cc:69]: How does versioning work? Is this line indicating that this operator is available in versions 6 - 12? Is this right? Should it just be available in future versions?

**Review by jamill — APPROVED** [review]
Maybe include motivation for why you are adding the double support for ceil?

**Comment by WilBrady** [comment]
Putting this on hold, while I investigate if this is blocking our test efforts. Based on feedback on previous and similar PR, I need a good justification for this change - https://github.com/microsoft/onnxruntime/pull/8708#issuecomment-898422359

File: onnxruntime/core/providers/cpu/cpu_execution_provider.cc (hunk 2 above)
Comment (WilBrady) [reply onnxruntime/core/providers/cpu/cpu_execution_provider.cc:69]: public docs link to release branches of the OperatorKernels.md which list ORT operator support.
https://onnxruntime.ai/docs/reference/operators/OperatorKernels.html

**Comment by WilBrady** [comment]
No compelling reason beyond some corner case print scenarios, so abandoning for now.

**Pull request closed without merging**